### ISSUE TITLE
higher-level function needed in adwin.py

### ISSUE THREAD

**Issue by light-matters** [vukics/Wigner_Time#32]
Currently, the notebooks still have to manually chain functions together to do necessary conversions.

There should be a super function that collects these together for convenience.

**Comment by light-matters** [comment]
Are you already using `to_adwin` or is this the function that you're looking for?

**Comment by vukics** [comment]
No, I think at the moment we use `adwin.add` and `adwin.output`.

**Comment by light-matters** [comment]
@vukics If you give me a list of anything else that  you need in this function, then I can update `to_adwin` with what's missing.

**Comment by vukics** [comment]
You tell me what I need :)

`to_adwin` should convert a timeline to ADwin output, that’s what I need :)

**Comment by light-matters** [comment]
Lol, well fair enough.

 I do consider you the high-wizard of ADwin though, so once PR-ed, you can let me know how it fares in the real world.

**Comment by light-matters** [comment]
`adwin.to_data` is currently the function for this.
